readme.md

```diff
@@ -574,6 +574,20 @@ is.all(is.string, '🦄', [], 'unicorns');
 //=> false
 ```
 
+##### .validDate(value)
+
+Returns `true` if the value is a valid date.
+
+`Invalid Date`s occur when an invalid value is passed to the `Date` constructor. This can be an invalid date string, a non-numeric value or a number that is outside of the expected range for a date value.
```

Comment:
Should more clearly describe that a `Date` instance can be in an invalid state, meaning it's possible to create an invalid `Date` without the constructor throwing. I don't think that's immediately clear to all users.